Jenkinsfile_CNP

```diff
@@ -67,23 +67,23 @@ withPipeline(type, product, component) {
   }
 
   before('smoketest:preview') {
-    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.preview.platform.hmcts.net/"
+    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.service.core-compute-preview.internal/"
```

Comment:
this url should be - https://prl-citizen-frontend-pr-886.preview.platform.hmcts.net/

So line change 70 is correct